src/main/java/net/imagej/ops/image/ImageNamespace.java

```diff
@@ -441,6 +444,27 @@
 		return result;
 	}
 
+	// -- coordinate equation --
```

Comment:
These should be located next to the other `equation(...)` methods under the `// -- equation --` heading.